app/flatpak-builtins-info.c

```diff
@@ -187,13 +190,24 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
           parent = ostree_commit_get_parent (commit_v);
           timestamp = ostree_commit_get_timestamp (commit_v);
           formatted_timestamp = format_timestamp (timestamp);
+
+          commit_metadata = g_variant_get_child_value (commit_v, 0);
+          g_variant_lookup (commit_metadata, "xa.metadata", "&s", &xa_metadata);
+          if (xa_metadata == NULL)
+            return flatpak_fail (error, "Commit has no metadata");
```

Comment:
The xa.metadata field was added in 0.9.4, and e.g. debian stable is still on 0.8.x, so I'm not sure we want to enforce this yet.

Reply:
We might want to change `builtins-info-remote.c` as well then, since it currently enforces `xa.metadata`.

Reply:
Also, `xa_metadata` is unused here anyway. The `collection_id` is (correctly) being loaded from `commit_metadata`.